module-api/src/main/java/com/yapp/sharefood/like/controller/LikeController.java

```diff
@@ -0,0 +1,41 @@
+package com.yapp.sharefood.like.controller;
+
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+
+@RestController
+public class LikeController {
+
+    @PostMapping("api/v1/food/{foodId}/like")
+    @ApiResponses({
+            @ApiResponse(code = 201, message = "[success] like 추가 성공"),
+            @ApiResponse(code = 409, message = "[error] 이미 like가 추가된 상태입니다.", response = HttpClientErrorException.Conflict.class)
+    })
+    public ResponseEntity<Void> createLike(@PathVariable("foodId") Long foodId) {
```

Comment:
Request Body와 User 정보를 가져오는 annotation도 사용해서 넣어주세요

```suggestion
    public ResponseEntity<Void> createLike(@AuthUser User user, @PathVariable("foodId") Long foodId) {
```